### PR TITLE
Updating top-level README 

### PR DESCRIPTION
**Pull request by sumitsumit**
* updated title
* rewrote initial text
* added links for each directory and updated explanation
* added citation section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,28 @@
-# Aurora-BP Study Data
+# The Aurora-BP Dataset and Study
 
-Here, you can access data related to the associated Aurora-BP Study publication: "[A Comparison of Wearable Tonometry, Photoplethysmography, and Electrocardiography for Cuffless Measurement of Blood Pressure in an Ambulatory Setting.](https://ieeexplore.ieee.org/document/9721156)"
+This repository contains sample code, sample data, and explanatory information for working with the **Aurora-BP dataset** published alongside the publication of the **Aurora-BP study**, i.e., Mieloszyk, Rebecca, et al. "[A Comparison of Wearable Tonometry, Photoplethysmography, and Electrocardiography for Cuffless Measurement of Blood Pressure in an Ambulatory Setting.](https://ieeexplore.ieee.org/document/9721156)" *IEEE Journal of Biomedical and Health Informatics* (2022). The dataset includes de-identified participant information, raw sensor data aligned with each measurement, and a wide variety of features derived from sensor data. The publishing of this dataset as well as the characterization of multiple feature groups across a broad population and multiple settings are intended to aid future cardiovascular research.
```
Comment (jeremiahwander) [inline README.md]: 'published alongside the publication' feels redundant

**Review by jeremiahwander — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
+- Mieloszyk, Rebecca, et al. "[A Comparison of Wearable Tonometry, Photoplethysmography, and Electrocardiography for Cuffless Measurement of Blood Pressure in an Ambulatory Setting.](https://ieeexplore.ieee.org/document/9721156)" *IEEE Journal of Biomedical and Health Informatics* (2022)
+
 ---
 
 ## Data Access
 
 ### Data Access Committee
 
-Requests for [data access](docs/README.md) are reviewed by the Data Access Committee. During review, submitting investigator and primary investigator may be contacted for verification. For further information regarding data access, contact: aurorabp@microsoft.com
+Requests for [data access](docs/README.md) are reviewed by the Data Access Committee. During review, the submitting investigator and primary investigator may be contacted for verification. The information you will need to gather for a submitting a Data Use Application as well as a link to the form are listed below. For additional questions regarding data access, contact: aurorabp@microsoft.com
```
Comment (twede) [inline README.md]: instead of "for a submitting" it should be "to submit" or "for a Data Use Application submission"

**Comment by gabe-microsoft** [comment]
I'm approving this as is, but I have a number of small fixes I want to make (orthogonal to the changes in this PR), so I'm going to make another PR for my fixes

**Review by gabe-microsoft — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (twede) [reply README.md]: maybe 
"...working with the **Aurora-BP Study dataset** as described in the **Aurora-BP study** publication: Mieloszyk, Rebecca, ..."

**Review by twede — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sumitsumit) [reply README.md]: perhaps 'released alongside the publication'?

File: README.md (hunk 2 above)
Comment (sumitsumit) [reply README.md]: good catch, fixing

**Comment by sumitsumit** [comment]
Made updates addressing Miah and Hope's feedback

File: README.md [hunk 3]
```diff
@@ -1,24 +1,28 @@
-# Aurora-BP Study Data
+# The Aurora-BP Dataset and Study
```
Comment (beckyatmsft) [inline README.md]: An ordering of "The Aurora-BP Study and Dataset" seems to make more chronological sense. Did you have a preference for this order ("... Dataset and Study") instead?

**Review by beckyatmsft — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (sumitsumit) [reply README.md]: sure, sounds good to me! :) I'll make the change and push

**Pull request merged**